### PR TITLE
various fixes for visibility, additional maxlength checks

### PR DESCRIPTION
**Pull request by Camelia-Orcid**

### CODE REVIEW

**Comment by Camelia-Orcid** [comment]
[[QA][my-orcid] Show account visibility setting in bio modal](https://trello.com/c/hjHvaGc7/7745-qamy-orcid-show-account-visibility-setting-in-bio-modal)

**Comment by Camelia-Orcid** [comment]
[[QA][my-orcid] Add whitespace before language of translated title in work details](https://trello.com/c/QERwkywl/7757-qamy-orcid-add-whitespace-before-language-of-translated-title-in-work-details)

**Comment by Camelia-Orcid** [comment]
[[QA][my-orcid] No error message in works modal when max character limit is exceeded](https://trello.com/c/WI1eYc2l/7750-qamy-orcid-no-error-message-in-works-modal-when-max-character-limit-is-exceeded)

File: src/app/record/components/funding/funding.component.html [hunk 1]
```diff
@@ -135,8 +135,8 @@ <h3 class="orc-font-body-small" i18n="@@funding.amount">
         <span class="translated-title-label orc-font-body-small">
           <ng-container i18n="@@funding.translatedTitle"
             >Translated title</ng-container
-          >
-        </span>
+          > </span
+        >&nbsp;
```
Comment (leomendoza123) [inline src/app/record/components/funding/funding.component.html:139]: On the HTML we have avoided using this `&nbsp` before. Instead, we have added `preserWhiteSpaces` on the component configurations. 

Just competing this so you know is an alternative option.

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**